### PR TITLE
Jnci 05162018

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@sidsharma27 Kindly review and merge if approved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,515 @@
...
+}
+```
+
+#### Method 2 - Manually copy the SDK files
+
+1. Clone this repository.
+2. Download the Agora Video SDK from [Agora.io SDK](https://www.agora.io/en/download/).
+3. Unzip the downloaded SDK package.
+4. Copy the .jar file from the **libs** folder of the downloaded SDK package to the **/apps/libs** folder of the sample application.
+5. Copy the .so files from the **arm64-v8a** folder of the downloaded SDK package to the **/app/src/main/jniLibs/arm64-v8a** folder of the sample application.
```
Comment (guohai) [inline README.md]: arm64-v8a or x86 is optional for developers.

File: README.md [hunk 2]
```diff
@@ -1,63 +1,515 @@
...
-*其他语言版本： [简体中文](README.zh.md)*
+This tutorial enables you to quickly get started in your development efforts to create an Android app with real-time video calls, voice calls, and interactive broadcasting. With this sample app you can:
 
-The Agora Android Tutorial 1to1 Sample App is an open-source demo that will help you get video chat integrated directly into your Android applications using the Agora Video SDK.
+* Join and leave a channel.
+* Mute and unmute audio.
+* Enable or disable video.
+* Choose between the front or rear camera.
 
-With this sample app, you can:
```
Comment (guohai) [inline README.md]: Also need some reference link to other platforms, such as iOS/macOS/Windows/Linux

File: README.md [hunk 3]
```diff
@@ -1,63 +1,515 @@
-# Agora Android Tutorial - 1to1
```
Comment (guohai) [inline README.md]: tutorial/sample/demo these words should defined clearly

File: README.md [hunk 4]
```diff
@@ -1,63 +1,515 @@
...
+1. Open the sample application in Android Studio.
+2. Build and run the sample project. This should display the application on your device.
+
+## Steps to Create the Sample 
+
+* [Set Permission](#set-permissions)
+* [Create Visual Assets](#create-visual-assets)
+* [Design the User Interface](#design-the-user-interface)
+
+For details about the APIs used to develop this sample, see the [Agora.io Documentation](https://docs.agora.io/en/2.2).
```
Comment (guohai) [inline README.md]: Reference the specific version, can we always use latest SDK version.

File: README.md [hunk 5]
```diff
@@ -1,63 +1,515 @@
...
+
+#### Method 2 - Manually copy the SDK files
+
+1. Clone this repository.
+2. Download the Agora Video SDK from [Agora.io SDK](https://www.agora.io/en/download/).
+3. Unzip the downloaded SDK package.
+4. Copy the .jar file from the **libs** folder of the downloaded SDK package to the **/apps/libs** folder of the sample application.
+5. Copy the .so files from the **arm64-v8a** folder of the downloaded SDK package to the **/app/src/main/jniLibs/arm64-v8a** folder of the sample application.
+6. Copy the .so files from the **armeabi-v7a** folder of the downloaded SDK package to the **/app/src/main/jniLibs/armeabi-v7a** folder of the sample application.
+7. Copy the .so files from the **x86** folder of the downloaded SDK package to the **/app/src/main/jniLibs/x86a** folder of the sample application.
```
Comment (guohai) [inline README.md]: x86a should be replaced with x86

File: README.md [hunk 6]
```diff
@@ -1,63 +1,515 @@
...
+
+```java
+private void joinChannel() {
+    mRtcEngine.joinChannel(null, "demoChannel1", "Extra Optional Data", 0); 
+}
+```
+
+The `channelName` parameter receives the name of the channel to join (`demoChannel1`). The call to [RtcEngine.joinChannel()](https://docs.agora.io/en/2.2/product/Video/API%20Reference/communication_android_video?platform=Android) enables the speakerphone when using Agora.
+
+**Note**: Users in the same channel can talk to each other, but users with different app IDs cannot call each other even if they join the same channel.
```
Comment (guohai) [inline README.md]: use App ID?

File: README.md (hunk 3 above)
Comment (jamesneimanconsulting) [reply README.md]: A tutorial is a set of instructions that guide the developer through the process.
A sample is a code sample.
A demo app is a synonym to a code sample. 
I think these terms are well understood by native English speaking developers. If these definitions should be included somewhere or you would prefer that some terms be changed please let me know.

**Comment by jamesneimanconsulting** [comment]
@guohai I have uploaded changes to this branch in a commit just now.

**Pull request merged**